src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Text } from '../Text';
+import styles from './Toggle.scss';
+
+export interface Props extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'type'> {
+    /** A unique id to reference this toggle */
+    id: string;
+    /** Label related to toggle */
+    children?: string;
+    /** Should be Toggle disabled or not */
+    disabled?: boolean;
+    /** Ref to access the toggle input element */
+    ref?: React.RefObject<HTMLInputElement>;
+}
+
+const { block, elem } = bem('Toggle', styles);
+
+export const Toggle: React.FC<Props> = React.forwardRef((props, ref) => {
+    const { children, disabled, id, ...rest } = props;
+    return (
+        <label {...block(props)} htmlFor={id}>
```

Comment:
It is minor, but we can rid of `htmlFor`, because the input is explicitly associated with the label when it inside the label.

Reply:
without this we have an eslint error